app/src/pages/admin/allUsersPage/usersToolbar/usersEntities.jsx

```diff
@@ -100,6 +100,11 @@ export class UsersEntities extends Component {
         title: intl.formatMessage(messages.name),
         active: true,
         removable: false,
+        customProps: {
```

Comment:
Why not `maxlength: 256,`?

Reply:
I did it similar to other fields (line 219)

Reply:
That input has different interface because of the original input is incapsulated by the underlying library.